### ISSUE TITLE
Update manager doesn't work

### ISSUE THREAD

**Issue by shadow2560** [shadow2560/Ultimate-Switch-Hack-Script#4]
The update manager is broken for now because of Github's Subversion support was stoped in january.

For now I have explored the solution to use he Github's API but it's too limited so I exclude it. I've also tested to upload the project to a web server but downloading some files doesn't want to work.

I'll continue to search a solution, if you have one to propose witch not costing me some money don't hesitate to propose it here.

**Comment by 3kynox** [comment]
Same problem for me, getting this while updating/installing modules

```
svn: E160013: '/shadow2560/Ultimate-Switch-Hack-Script/trunk/tools/default_configs' path not found
```

**Comment by shadow2560** [comment]
> Same problem for me, getting this while updating/installing modules
> 
> ```
> svn: E160013: '/shadow2560/Ultimate-Switch-Hack-Script/trunk/tools/default_configs' path not found
> ```

Yes, it's because Github doesn't allow SVN anymore, I'm trying to find a fix.

**Comment by shadow2560** [comment]
Fixed in version 4.3.318, the previous versions should also work normaly.
